sympy/stats/frv_types.py

```diff
@@ -342,6 +335,12 @@ def Coin(name, p=S.Half):
     >>> density(C2).dict
     {H: 3/5, T: 2/5}
 
+    Parameters
+    ==========
+
+    p : Rational Numeber between 0 and 1
+      Represents probability of getting "Heads", by default is Half
```

Comment:
Numeber should be Number. I think defaults should be specified on the first line like:
```
    p : real number between 0 and 1 (inclusive), default 1/2
        Probability of getting "Heads"
```